policyengine_us/variables/gov/states/me/adjusted_gross_income/me_military_retirement_pay_subtractions.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class me_military_retirement_pay_subtractions(Variable):
```

Comment:
Since this isn't Maine-specific could you move this to `household/income/person/pensions/military_retirement_pay.py`?